runners/lock_helper.py

```diff
@@ -74,13 +71,12 @@ def generate_identifier():
     return u"{0}-{1}".format(socket.gethostname(), uuid.uuid4().hex)
 
 
-def get_zookeeper_client(zookeeper_hosts=None):
-    if zookeeper_hosts is not None:
-        host_string = u",".join(zookeeper_hosts)
+def get_zookeeper_client(zookeeper_hosts):
+    # kazoo requires a comma separated string.
+    if isinstance(zookeeper_hosts, list):
+        host_string = ",".join(zookeeper_hosts)
     else:
```

Comment:
Here we assume that the only other kind of value we'll receive is a string. I guess that's ok, but we could make the check more explicit if you'd like.